specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-11-30-preview/definitions/MigrateMySqlAzureDbForMySqlOfflineTask.json

```diff
@@ -0,0 +1,500 @@
+          "description": "Migration status message",
+          "readOnly": true
+        },
+        "message": {
+          "type": "string",
+          "description": "Migration progress message",
+          "readOnly": true
+        },
+        "databases": {
+          "type": "string",
+          "description": "Selected databases as a map from database name to database id",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "databaseSummary": {
+          "type": "string",
+          "description": "Summary of database results in the migration",
+          "additionalProperties": {
+            "$ref": "#/definitions/DatabaseSummaryResult"
+          },
+          "readOnly": true
+        },
+        "migrationReportResult": {
+          "type": "object",
+          "description": "Migration Report Result, provides unique url for downloading your migration report.",
+          "$ref": "./MigrationValidation.json#/definitions/MigrationReportResult"
+        },
+        "sourceServerVersion": {
+          "type": "string",
+          "description": "Source server version",
+          "readOnly": true
+        },
+        "sourceServerBrandVersion": {
+          "type": "string",
+          "description": "Source server brand version",
+          "readOnly": true
+        },
+        "targetServerVersion": {
+          "type": "string",
+          "description": "Target server version",
+          "readOnly": true
+        },
+        "targetServerBrandVersion": {
+          "type": "string",
+          "description": "Target server brand version",
+          "readOnly": true
+        },
+        "exceptionsAndWarnings": {
+          "description": "Migration exceptions and warnings.",
+          "type": "array",
+          "items": {
+            "$ref": "./Common.json#/definitions/ReportableException"
+          },
+          "readOnly": true
+        },
+        "lastStorageUpdate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time the storage was updated",
+          "readOnly": true
+        },
+        "sourceBinlogPosition": {
+          "description": "The binlog position when the consistent snapshot was taken",
+          "$ref": "./MigrateMySqlCommon.json#/definitions/MySqlBinlogPositionOutput",
+          "readOnly": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/MigrateMySqlAzureDbForMySqlOfflineTaskOutput"
+        }
+      ]
+    },
+    "MigrateMySqlAzureDbForMySqlOfflineTaskOutputDatabaseLevel": {
+      "type": "object",
+      "x-ms-discriminator-value": "DatabaseLevelOutput",
+      "properties": {
+        "databaseName": {
+          "type": "string",
+          "description": "Name of the database",
+          "readOnly": true
+        },
+        "startedOn": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Migration start time",
```

Comment:
Is this uber level Migration start time or start time for this database? Should we describe it appropriately? As currently it is described the same way above as well in MigrationLevelOutput

Reply:
This is existing code. I will change it anyways.